frontendIcons/src/main/resources/pluginIcon.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hm we now have pluginIcon.svg both in top-level `src/` and in `frontendIcons/src/`... is this duplication necessary?

Reply:
actually they do differ. `frontendIcons`-ish one renamed to macheteFileIcon.svg